app/helpers/profile_helper.rb

```diff
@@ -0,0 +1,22 @@
+#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
+#   licensed under the Affero General Public License version 3 or later.  See
+#   the COPYRIGHT file.
+
+module ProfileHelper
+  def upper_limit_date_of_birth()
+     minimum_year=AppConfig.age.minimum
```

Comment:
The key is `AppConfig.settings.terms.minium_age` Also please use spaces around the `=` in assignments.